cmake/SociConfig.cmake

```diff
@@ -69,7 +69,7 @@ else()
             set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
         elseif (CMAKE_SYSTEM_NAME MATCHES "Windows")
             # We need to disable the warnings for using %I64d with MinGW.
-            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format")
+            set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wno-pedantic-ms-format -Wno-format")
```

Comment:
@vadz could you please check this log https://ci.appveyor.com/project/mloskot/soci/branch/master/job/0cc8sb7ttjkbyegr ? What do you think? This code already have ```-Wno-pedantic-ms-format```